firestore/tests/unit/v1/test_client.py

```diff
@@ -118,8 +118,8 @@ def test__firestore_api_property_with_emulator(
         self.assertIs(firestore_api, mock_client.return_value)
         self.assertIs(firestore_api, client._firestore_api_internal)
 
-        self.assertEqual(mock_insecure_channel.call_count, 1)
-        self.assertEqual(mock_insecure_channel.call_args.args[0], emulator_host)
+        mock_insecure_channel.assert_called_once_with(emulator_host)
+
```

Comment:
Extra newline?  Anyway, `lint` is failing, needs blackening.

Reply:
Thanks. Will merge on green.